package.json

```diff
@@ -24,23 +24,35 @@
   "license": "BSD-3-Clause",
   "homepage": "https://github.com/SalesforceCommerceCloud/sfcc-ci#readme",
   "dependencies": {
+    "archiver": "^5.0.0",
```

Comment:
@jordanebachelet I cannot find this NPM module being used. Can you please remove and check with the other added? We should only add ones, which are actually used.

Reply:
Used here sir: https://github.com/SalesforceCommerceCloud/sfcc-ci/pull/174/files#diff-3e98ae93aa4f2aad8953d2195129cc5a65502df35f4ede3f254a429a3729cbebR310